src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_data/v8_check_trap.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Fatal error in foo.cc, line 4711
+# Check failed: 3 == 2
+#
+AddressSanitizer:DEADLYSIGNAL
+=================================================================
+==1450490==ERROR: AddressSanitizer: TRAP on unknown address 0x000000000000 (pc 0x7ff87c68081b bp 0x7ffd59cb1bf0 sp 0x7ffd59cb1be0 T0)
+SUMMARY: AddressSanitizer: TRAP (/foobar/libv8_libbase.so+0x4d81b)
+==1450490==ABORTING
+
```

Comment:
can you add some stackframes here for a more realistic test?

Reply:
Ack, I copied in a complete stack trace now.